app/code/Magento/InventoryCatalog/etc/module.xml

```diff
@@ -8,7 +8,7 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
     <module name="Magento_InventoryCatalog" setup_version="2.0.0">
         <sequence>
-            <module name="InventorySales"/>
+            <module name="Magento_Inventory"/>
```

Comment:
we should have a dependency on `Magento_InventoryAPI` but not on Inventory

Reply:
It is not dependency
But it is sequence, so we need already installed tables which are described in Magento_Inventory